executor/app.py

```
@@ -0,0 +1,504 @@
+async def close_all_claude_sessions():
+    """
+    Close all Claude client connections
+    """
+    try:
+        status, message = await agent_service.close_all_claude_sessions()
+        if status == TaskStatus.SUCCESS:
+            return {"message": message}
+        else:
+            raise HTTPException(status_code=500, detail=message)
+    except Exception as e:
+        logger.exception(f"Error closing all Claude client connections: {str(e)}")
```

Ruff (0.14.14)

Use explicit conversion flag

Replace with conversion flag

(RUF010)